pkg/zstd/gen.lua

```diff
@@ -39,14 +39,15 @@ lib('libzstd.a', [[lib/(
 	)
 	decompress/(
 		huf_decompress.c
+		huf_decompress_amd64.S
```

Comment:
```suggestion
		@x86_64 huf_decompress_amd64.S
```

This way this won't break when compiling on aarch64.

Reply:
(I have tested my suggestion on aarch64 and can confirm zstd builds correctly)

Reply:
looks good. applied.